samza-test/src/test/scala/org/apache/samza/test/integration/StreamTaskTestUtil.scala

```diff
@@ -133,43 +130,36 @@ object StreamTaskTestUtil {
     config.put(ProducerConfig.LINGER_MS_CONFIG, "0")
     val producerConfig = new KafkaProducerConfig("kafka", "i001", config)
 
+    adminClient = AdminClient.create(config.asInstanceOf[util.Map[String, Object]])
     producer = new KafkaProducer[Array[Byte], Array[Byte]](producerConfig.getProducerProperties)
 
     createTopics
     validateTopics
   }
 
   def createTopics {
-    AdminUtils.createTopic(
-      zkUtils,
-      INPUT_TOPIC,
-      TOTAL_TASK_NAMES,
-      REPLICATION_FACTOR)
+    adminClient.createTopics(Collections.singleton(new NewTopic(INPUT_TOPIC, TOTAL_TASK_NAMES, REPLICATION_FACTOR.shortValue())))
   }
 
   def validateTopics {
```

Comment:
Not sure if this is feasible/practical to combine, but it looks like some of this code is similar to code from `TestKafkaSystemAdmin`, such as `validateTopics`.

Reply:
It is doable and I considered it. I deferred it due to following

1. I didn't want to add more churn to scala code. StreamTaskTestUtil still uses some of the TestUtils deprecated code and I chose to clean up all at once as a separate PR (maybe rewrite them in java)
2. I didn't want to expand the scope of this PR.

I have created a follow up ticket (https://issues.apache.org/jira/browse/SAMZA-2145) to clean up the test and consolidate the common code to some helper or move it to test harness.